locales-pending/landing-all.ftl

```diff
@@ -0,0 +1,11 @@
+public-nav-name = { -brand-fx-monitor }
```

Comment:
Should this file live in `/locales`, since you're calling out it's for international users?

Reply:
Yes, eventually, that's why I put them in separate files for now. However, I'm not yet confident that the strings will no longer changes, so I don't want to create extra worker for the localisers. And more strings will still be added, so I also want to reduce the number of Pontoon notifications they'll get.